docs/how-to/use-audit-logs/debug-mode.mdx

```diff
@@ -54,4 +55,17 @@ Another option to enable Debug Mode is through Permit's UI. To do that:
 1. Go to your Permit dashboard and make sure you have the relevant project and environment selected.
 2. Next, go to the **Audit Log** screen.
 3. In the top right corner, click on **Settings**.
-4. Click the toggle to enable Debug Mode.
+4. Click the toggle to change the default Debug Mode behavior on the PDPs in this environment.
+
+
+
+:::note Default Behavior
+The default behavior for the debug mode is "unset" .
+If the debug mode was not changed from unset, it will be treated as activated
+:::
+
+:::warning Disadvantages
```

Comment:
Maybe make the title less negative? E.g. "performance"?

Reply:
Changed to Performance Impact